crates/runtime/src/lib.rs

```diff
@@ -279,6 +282,32 @@ impl Runtime {
         Ok(())
     }
 
+    pub async fn load_models(&self) {
+        for model in self.app.read().await.models.clone() {
```

Comment:
If we're only passing a reference to a Model, do we need to clone it?

Reply:
Updated, thank you!